ckanext/ontario_theme/i18n/ckanext-ontario_theme.pot

```diff
@@ -1933,3 +1933,35 @@ msgstr ""
 #: ckanext/ontario_theme/page.py:51
 msgid "Go to page {}"
 msgstr ""
+
+#: ckanext/ontario_theme/templates/internal/header.html:115
+msgid "Navigation header"
+msgstr ""
+
+#: ckanext/ontario_theme/templates/internal/header.html:152
+msgid "Clear field"
+msgstr ""
+
+#: ckanext/ontario_theme/templates/internal/header.html:170
+msgid "Search"
+msgstr ""
+
+#: ckanext/ontario_theme/templates/internal/header.html:182
+msgid "close search bar"
```

Comment:
Really minor nitpick, but this should have a capital c.

Reply:
I swear I did change it, will make that change again with any of your other comments.

Reply:
Everything else looks good, this is the only thing that requires change